Makefile

```diff
@@ -241,15 +241,21 @@ install-resources:
 
 IS_HOSTED ?= false
 E2E_PROCS = 20
+TEST_NAMESAPCE = managed
 
 .PHONY: e2e-test
 e2e-test: e2e-dependencies
-	$(GINKGO) -v --procs=$(E2E_PROCS) $(E2E_TEST_ARGS) test/e2e -- -is_hosted=$(IS_HOSTED)
+	$(GINKGO) -v --procs=$(E2E_PROCS) $(E2E_TEST_ARGS) test/e2e -- -is_hosted=$(IS_HOSTED) -test_namespace=$(TEST_NAMESAPCE)
 
 .PHONY: e2e-test-coverage
 e2e-test-coverage: E2E_TEST_ARGS = --json-report=report_e2e.json --label-filter='!hosted-mode && !running-in-cluster' --output-dir=.
 e2e-test-coverage: e2e-run-instrumented e2e-test e2e-stop-instrumented
 
+.PHONY: e2e-ocm-ns-test-coverage
+e2e-ocm-ns-test-coverage: E2E_TEST_ARGS = --json-report=report_e2e.json --label-filter='ocm-ns' --output-dir=.
```

Comment:
I don't think you need to separate the tests. Simplify having `TEST_NAMESPACE=managed` will cause it to watch the `managed` and `open-cluster-management-policies` namespaces which is what users will experience when it's installed as part of OCM and ACM.

The user could explicitly set the watch namespace to `open-cluster-management-policies` when the config-policy-controller is manually deployed, but we don't need to explicitly test that since that's not a different code path.